### PR TITLE
Updated data source definition for atlassian-jira

### PR DESCRIPTION
**Pull request by bshankararamaya**
Updated data source definition for atlassian-jira

marketplace PR: https://source.datanerd.us/nr1-dev-experience/marketplace/pull/592
Data source id: atlassian-jira

Ticket: https://issues.newrelic.com/browse/NR-148884

To Release this feature , adding nerdlet state to this data source.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by pkosanam — APPROVED** [review]
Here are the below checks
dataSourceId "atlassian-jira"

Quickstart https://newrelic.com/instant-observability/atlassian-jira is matching the datasource ID
dataSourceId is matching the ID present in above mentioned marketplace PR
Hence, it looks good

**Review by mdumpati — DISMISSED** [review]
LGTM

File: quickstarts/atlassian/atlassian-jira/config.yml
Comment (mdumpati) [inline quickstarts/atlassian/atlassian-jira/config.yml:1]: @bshankararamaya This Datasource ID another PR got merged https://github.com/newrelic/newrelic-quickstarts/pull/1927 so can you check with @pkosanam on this and do the changes only nerdlet and datasource

**Review by mdumpati — APPROVED** [review]
LGTM

File: quickstarts/atlassian/atlassian-jira/config.yml
Comment (pkosanam) [reply quickstarts/atlassian/atlassian-jira/config.yml:1]: Observed that @bshankararamaya has made necessary changes. Hence, marking this conversation as "resolved"

**Review by pkosanam — APPROVED** [review]
LGTM

File: quickstarts/atlassian/atlassian-jira/config.yml
Comment (bshankararamaya) [reply quickstarts/atlassian/atlassian-jira/config.yml:1]: updated as per suggestions

**Pull request merged**